Bio/SeqIO/InsdcIO.py

```diff
@@ -1117,6 +1118,17 @@ def write_record(self, record):
         handle.write("FEATURES             Location/Qualifiers\n")
         rec_length = len(record)
         for feature in record.features:
+            # Handle any stray line breaks in the feature
+            for k in feature.qualifiers.keys():
+                if (
+                    isinstance(feature.qualifiers[k], Iterable)
+                    and "\n" in feature.qualifiers[k]
+                ):
+                    feature.qualifiers[k] = feature.qualifiers[k].replace("\n", " ")
+                    warnings.warn(
+                        "Replacing stray line break with space",
```

Comment:
I would at least include the annotation qualifier key in this message, and perhaps also the ``record.id``.

We might also include the index of the feature (using ``enumerate(...)`` to loop over the feature list).